### ISSUE TITLE
Slight background changes

### ISSUE THREAD

**Issue by bunniewormy** [transhumandesign/kag-base#365]
Can we please add back the dope background from kag classic?
Current kag:
![screen-19-01-21-00-17-56](https://user-images.githubusercontent.com/39147836/51446424-34904e00-1d12-11e9-92f4-e47fcd5fcc7e.png)
![image](https://user-images.githubusercontent.com/39147836/51446566-2d6a3f80-1d14-11e9-8d14-222b83bc761f.png)
Meanwhile, kag classic:
![image](https://user-images.githubusercontent.com/39147836/51446516-7d94d200-1d13-11e9-8fe3-67f8de37d7cf.png)
![image](https://user-images.githubusercontent.com/39147836/51446602-b71a0d00-1d14-11e9-81c3-00a17f77faf8.png)
![image](https://user-images.githubusercontent.com/39147836/51446603-cac57380-1d14-11e9-83d8-ea4d171302dd.png)

The current background just feels empty and sad. Imo, we should have all the clouds from Classic and the fog-like clouds [on ground level] back, as well as the additional layers back (1 more in the front and 1 more in the background i think? mainly thinking about the image below). Also, the background should be moved up - less sky should be visible, and more background.
![backgroundcastle](https://user-images.githubusercontent.com/39147836/51446545-d7959780-1d13-11e9-9ab9-1b33c990cc8e.png)

**Comment by Franrek** [comment]
The could could be reworked but the rest does look richer in alpha.

**Comment by Radinyn** [comment]
The current background just feels empty and sad. Imo, we should have all the clouds from Classic and the fog-like clouds [on ground level] back, as well as the additional layers back (1 more in the front and 1 more in the background i think? mainly thinking about the image below). Also, the background should be moved up - less sky should be visible, and more background. hehe

**Comment by Shadowblitz16** [comment]
I would even be for adding biome specific backgrounds but ya I agree with this

**Comment by bunniewormy** [comment]
Is this going to happen?

**Comment by CatWiz** [comment]
please make it happen

**Comment by AsuMagic** [comment]
![mockup](https://user-images.githubusercontent.com/8986152/85211092-fe5f5f80-b345-11ea-8130-5a07c3200097.png)
Did an early mockup of what it could look like. This was made in an image editor, not ingame.  
Not quite sure about the sky color and cloud opacity, but this can be of course be tweaked when implemented.  
Sprites were upscaled using scale2x so that it looks better in that higher resolution mockup.

I think the proper approach would be to implement it script-side, which is definitely feasible but probably will be a bit time consuming.

**Comment by Vam-Jam** [comment]
![image](https://user-images.githubusercontent.com/44256866/93820999-d984cc00-fc55-11ea-96d1-db0e7306c106.png)

https://github.com/TFlippy/kag_territorycontrol/blob/master/TFlippy_TerritoryControl_Core_Dev/Gamemode/Rules/Scripts/Clouds.as

Script side implementation of the clouds is above, not final but its enough to let us play around with it.

There's a few things that still need doing and improving (read top of the file) before it should be added into vanilla.

TLDR of the script file;
- Clouds are synced, position and sprite
- Currently no fog on the ground
- All clouds done in one draw call (All clouds sit in one .png file)
- Clouds alpha will change based on day time (otherwise at night the cloud will be really bright)
